ComponentKit/Utilities/CKComponentAction.h

```diff
@@ -145,12 +139,14 @@ struct CKTypedComponentAction {
     const id responder = behavior == CKComponentActionSendBehaviorStartAtSender ? target : [target nextResponder];
     CKComponentActionSendResponderChain(_internal.selector(), responder, sender, args...);
   }
-
-  CKTypedComponentActionValue _internal;
 };
 
 typedef CKTypedComponentAction<> CKComponentAction;
 
+/** Explicit instantiation of our most commonly-used templates to avoid bloat in callsites. */
+extern template class CKTypedComponentAction<>;
+extern template class CKTypedComponentAction<id>;
```

Comment:
Here's the bit that forces the extern instantiation of our common action templates. We may want to add a few more here like UIEvent * and NSString * or whatever else people use. For now, this should do for the immediate conversion.